lib/components/securitySettingsToggles/securitySettingsToggles.js

```diff
@@ -0,0 +1,121 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  HourglassIcon,
+  WatermarkIcon,
+  ExpirationPicker,
+  ItemOption,
+  PersistentProtectionIcon,
+} from '@';
+
+import styles from './securitySettingsToggles.css';
+
+const SecuritySettingsToggles = ({
+  inModal,
+  showFilesSettings,
+  expirationDateTime,
+  watermark,
+  persistentProtection,
+  disabled,
+  onChange,
+}) => {
+  const [expirationChecked, setExpirationChecked] = useState(!!expirationDateTime);
+
+  const triggerExpirationDateTimeChange = useCallback(
+    (time) => {
+      onChange?.('expirationDateTime', time);
+    },
+    [onChange],
+  );
+
+  const expirationToggleChangeHandler = useCallback(
+    (checked) => {
+      setExpirationChecked(checked);
+
+      if (!checked) {
+        triggerExpirationDateTimeChange(null);
+      }
+    },
+    [triggerExpirationDateTimeChange],
+  );
+
+  const toggleChangeHandler = useCallback(
+    (name) => (checked) => {
+      onChange?.(name, checked);
+    },
+    [onChange],
+  );
+
+  return (
+    <>
+      <ItemOption
+        icon={<HourglassIcon />}
+        name="Expiration Date"
+        hasValue={expirationChecked}
+        disabled={disabled}
+        tooltipTitle="components.securitySettingsToggles.expiration.tooltip"
+        changeSwitch={expirationToggleChangeHandler}
+        placementTooltip="top"
+      />
+      {expirationChecked &&
+        (inModal ? (
+          <div className={styles.expirationPickerWrap}>
+            <ExpirationPicker
+              isModal
+              expDateTime={expirationDateTime?.toString()}
+              getTime={triggerExpirationDateTimeChange}
+            />
+          </div>
+        ) : (
+          <ExpirationPicker
+            expDateTime={expirationDateTime?.toString()}
+            getTime={triggerExpirationDateTimeChange}
+          />
+        ))}
+
+      {showFilesSettings && (
```

Comment:
I wonder if we should allow applications to decide which controls should be shown and enabled, etc.

Any thoughts here @jrschumacher?

Reply:
we use `showFilesSettings ` because the watermark and PFP can be set up only for files.